src/openforms/submissions/tests/test_submission_suspension.py

```diff
@@ -32,6 +33,11 @@
 
 
 class SubmissionSuspensionTests(SubmissionsMixin, APITestCase):
+    def setUp(self):
+        # clear caches to avoid problems with throttling; @override_settings not working
+        # see https://github.com/encode/django-rest-framework/issues/6030
+        self.addCleanup(clear_caches)
```

Comment:
please also add the `super().setUp()` call, as mixins added to the test case may also perform some setup for every test.